ios/versioned/sdk43/EASClient/EASClient/EASClientModule.swift

```diff
@@ -6,8 +6,10 @@ public class EASClientModule: Module {
   public func definition() -> ModuleDefinition {
     name("EASClient")
 
-    constants([
-      "clientID": EASClientID.uuid().uuidString
-    ])
+    constants {
+      return [
+        "clientID": EASClientID.uuid().uuidString
+      ]
+    }
```

Comment:
Is this change to support an old version of expo-modules-core?

Reply:
Yep, the declaration of `constants` takes a function in the old version.